priv/repo/seeds.exs

```diff
@@ -153,6 +159,13 @@ defmodule CsGuide.Import do
     |> List.first()
     |> Enum.find_index(fn e -> e == name end)
   end
+
+  defp extract_postcode(str) do
+    postcode_regex =
```

Comment:
This should use the `Fields.Validate.postcode` function to keep things `DRY`.
But we can keep it for now in the interest of _shipping_.